src/seq/mod.rs

```diff
@@ -617,8 +607,9 @@ mod test {
             }
             for count in chosen.iter() {
                 // Samples should follow Binomial(1000, 1/9)
-                let err = *count - 1000 / 9;
-                assert!(-30 <= err && err <= 30);
+                // Octave: binopdf(x, 1000, 1/9) gives the prob of *count == x
+                // Note: have seen 153, which is unlikely but not impossible.
+                assert!(72 < *count && *count < 154, "count not close to 1000/9: {}", count);
```

Comment:
This seems fine. The old limits had no scientific analysis behind them. I mainly picked as small of a number as I could get away with.

Reply:
I was actually surprised I had to go this high — `binopdf(153, 1000, 1/9)` is 1.1e-5. But not impossible and I'm fairly sure the algorithm is right.